charts/airbyte/values.yaml

```diff
@@ -867,6 +868,9 @@ externalDatabase:
   database: db-airbyte
   port: 5432
 
+externalTemporal:
+  host: ""
+  port: ""
```

Comment:
Please read the file for reference and add the correct comment lines here. Maybe move the code more close to temporal.

Reply:
Good way is implement the same way as the external database.